packages/cli/config/eslint-factory.js

```diff
@@ -167,7 +167,11 @@ function createConfig(dir, extraConfig = {}) {
         },
       },
       {
-        files: ['**/src/generated/**/*.ts'],
+        files: [
+          // old location
+          '**/src/generated/**/*.ts',
+          '**/src/schema/openapi/generated/**/*.ts',
```

Comment:
I'm feeling this is a bit too specific. I'd be happy to switch to `**/src/**/generated/**/*.ts` tbh

Reply:
Missing this update still, and https://github.com/backstage/backstage/pull/23610/files#diff-49cd333ce926fe7319992883245939b7ff7edd25e34b3ec0a3e487fa660b500d

Reply:
😅 updated!